cluster/crds/pkg.crossplane.io_deploymentruntimeconfigs.yaml

```diff
@@ -4267,8 +4265,6 @@ spec:
                                         More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
                                         TODO: Add other useful fields. apiVersion, kind, uid?
                                       type: string
-                                  required:
```

Comment:
This one is also suspicious, I wouldn't expect it to change given the dependency this PR is bumping. I think this is the field: https://docs.crossplane.io/v1.16/api/#DeploymentRuntimeConfig-spec-deploymentTemplate-spec-template-spec-imagePullSecrets-name